src/components/ButtonLoading.tsx

```diff
@@ -1,16 +1,20 @@
 /* eslint-disable */
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 
 interface Props {
   style?: any;
 }
-const ButtonLoading: React.FC<Props> = ({ style }) => (
-  <div className={`flex btn w-fit py-1 px-3 bg-purple-500 font-bold ${style}`}>
-    <div className="loader mr-1" />
-    <button className="ml-1 text-white font-bold" disabled>
-      Processing...
-    </button>
-  </div>
-);
+const ButtonLoading: React.FC<Props> = ({ style }) => {
+  const { t } = useTranslation();
+  return (
+    <div className={`flex btn w-fit py-1 px-3 bg-primary font-bold ${style}`}>
+      <div className="loader mr-1" />
+      <button className="ml-1 text-white font-bold" disabled>
+        {t('Processing...')}
```

Comment:
Processing word does not make sense here. Something like `signing you in` is more relatable.